cccs-build/superset/analytical-platform-requirements.txt

```diff
@@ -0,0 +1 @@
+hogwarts-auth==1.8.0.7820
```

Comment:
Do we want to pin this precisely? Would we be better off with `1.8.*` or maybe using `=~`?

Reply:
We defiantly can, was just using the version I knew worked